appliances.yaml

```diff
@@ -1,6 +1,7 @@
 appliances:
   plex:
     name: "Plex"
+    url_name: "plex"
```

Comment:
Why do you still need this file? Is this used by the index page or something?

Reply:
it is used on the download pages... unless it is available some other way?